extensions/adobe/experience/adcloud/advertisementdetails.schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/adcloud/advertisementdetails",
+  "title": "Advertisement Details",
+  "type": "object",
+  "description": "Adobe AdCloud Ad Details",
+  "definitions": {
+    "advertisementDetails": {
+      "properties": {
+        "https://ns.adobe.com/experience/adcloud/advertisementdetails/adId": {
+          "title": "Ad ID",
+          "type": "integer",
```

Comment:
Why is this an integer and not a string?  Does this have anything to do with the AdID standard? if not, you need another name

Reply:
Changed type to string and added explicit differentiator from AdID in description. 

With respect to AdID standard, no this is not the same. This field is an AdCloud specific identifier for the Ad associated with the event. Given this is the global variable used throughout AdCloud services/reports and this is under AdCloud's extension I don't see any harm having an overlap in terminology with Ad-ID - not supported in AdCloud.